UITableView cells: use UITableViewCellStyleSubtitle for readability.

oryonatan opened the pull request:
Previously we used UITableViewCellStyleValue1 which causes long strings
to be truncated, the UITableViewCellStyleSubtitle style allows longer
strings to be shown.

barakwei commented on FBTweak/_FBActionTweakTableViewCell.m:
```diff
@@ -31,7 +31,7 @@ @implementation _FBActionTweakTableViewCell
 - (instancetype)initWithStyle:(UITableViewCellStyle)style
               reuseIdentifier:(NSString *)reuseIdentifier {
-  if ((self = [super initWithStyle:UITableViewCellStyleValue1 reuseIdentifier:reuseIdentifier])) {
+  if ((self = [super initWithStyle:UITableViewCellStyleSubtitle reuseIdentifier:reuseIdentifier])) {
```
don't change this one

barakwei commented on FBTweak/_FBTweakColorViewControllerHexDataSource.m:
```diff
@@ -77,7 +77,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 - (_FBEditableTweakTableViewCell *)hexCell
 {
-  _FBEditableTweakTableViewCell *hexCell = [[_FBEditableTweakTableViewCell alloc] initWithStyle:UITableViewCellStyleValue1
+  _FBEditableTweakTableViewCell *hexCell = [[_FBEditableTweakTableViewCell alloc] initWithStyle:UITableViewCellStyleSubtitle
```
don't change this one

barakwei commented:
Fix, squash and rebase. This repo desn't let you merge unless the branch is fully rebased.

oryonatan committed:
UITableView cells: use UITableViewCellStyleSubtitle for readbility.

Previously we used UITableViewCellStyleValue1 which causes long strings
to be truncated, the UITableViewCellStyleSubtitle style allows longer
strings to be shown.

oryonatan commented:
round

barakwei commented:
Fix commit msg & PR title.
